build-from-sources.yml

```diff
@@ -58,6 +58,10 @@ services:
     ports:
       - 443:5601
     environment:
+      # check only one node health for start kibana
+      - ELASTICSEARCH_URL=https://elasticsearch:9200
```

Comment:
in startup shell, you check ELASTICSEARCH_URL for elastic search i don't change that config and add
ELASTICSEARCH_HOSTS to allow kibana use multi elastic search nodes